corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -232,6 +233,18 @@ def to_report_filter(self, configuration, index):
         }
         if configuration['format'] == 'Date':
             filter.update({'compare_as_string': True})
+        if filter_format == 'dynamic_choice_list' and self._id == COMPUTED_OWNER_NAME_PROPERTY_ID:
+            filter.update({"choice_provider": {"type": "owner"}})
+        if filter_format == 'dynamic_choice_list' and self._id == COMPUTED_USER_NAME_PROPERTY_ID:
+            filter.update({"choice_provider": {"type": "user"}})
+        if filter_format == 'dynamic_choice_list' and self._id == COMPUTED_OWNER_LOCATION_PROPERTY_ID:
+            filter.update({"choice_provider": {"type": "location"}})
+        if configuration.get('pre_value') or configuration.get('pre_operator'):
+            filter.update({
+                'type': 'pre',  # type could have been "date"
+                'pre_operator': configuration.get('pre_operator', None),
+                'pre_value': configuration.get('pre_value', []),
+            })
```

Comment:
this diff is funny. i actually just moved the other two options from above to below these (easier if you review by commit)